esphome/esp32-s3-zero--deye8k.yaml

```
@@ -581,7 +581,14 @@ switch:
         - switch.turn_off: grid_charge
         - delay: 5s
         - button.press: use_solar_reference
-            
+  
+  - platform: template
+    optimistic: True
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)